### ISSUE TITLE
Convergence of GAN

### ISSUE THREAD

**Issue by Amirhossein-Esmaili** [caffe-mocha/dcgan-pytorch-cifar10#1]
Hello,

I used your code and I get the following training loss curve when using your code to train the DCGAN for CIFAR-10 for 25 epochs:

<img width="625" alt="Screen Shot 2021-12-17 at 11 37 26 AM" src="https://user-images.githubusercontent.com/33773340/146598753-847a9969-ff75-4f06-8f40-84f7c4b0a352.png">

I have tuned my hyperparameters and I am using 0.0002 for G_learning_rate and 0.002 for D_learning_rate (different learning rates for G and D). I also train G twice and D once in each iteration. This for me has resulted in the best results in terms of the training curve (better than when I use your code with the default parameters). However, I think my results do not converge, while the results you posted on your README page are converging (oscillating around a value with a relatively low/constant variance). For me, the oscillation of loss values increases over time. Do you have any idea or have encountered this before?
